README.md

```diff
@@ -218,6 +222,33 @@ map <C-n> :lne<CR>
 map <C-m> :lp<CR>
 ```
 
+### Using with Neovim
+
+Run `:GoRun` in a new tab, horizontal split or vertical split terminal
+
+```vim
+au FileType go nmap <leader>rt <Plug>(go-run-tab)
+au FileType go nmap <Leader>rs <Plug>(go-run-split)
+au FileType go nmap <Leader>rv <Plug>(go-run-vertical)
+```
+
+By default new terminals are opened in a vertical split. To change it
+
+```vim
+let g:go_term_mode = "split"                   "or set to tab
+```
+
+By default the testing commands run asynchronously in the background and display results
+with `go#jobcontrol#Statusline()`. To make them run in a new terminal
+
+```vim
+let g:go_term_enabled = 1
+```
+
+Add `go#jobcontrol#Statusline()` to your statusline for updates on running jobs
+
+![statusline](http://i.imgur.com/dNIef4Y.png)
```

Comment:
Can we remove this part and the sentence above? This is looks to big. I'll add something later with more context